maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,24 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+$title-decoration-position-top: -35px;
```

Comment:
If you are going to use this value only once, you dont need to create a constant for this one and left one.

Try to use `right` and `bottom` instead with smaller / positive numbers

Reply:
Change to `bottom` requires negative values too. Using `right` requires bigger value.